src/applications/vaos/utils/featureFlags.js

```diff
@@ -18,7 +18,7 @@ module.exports = [
   { name: 'vaOnlineSchedulingBreadcrumbUrlUpdate', value: true },
   { name: 'vaOnlineSchedulingBookingExclusion', value: false },
   { name: 'vaOnlineSchedulingStaticLandingPage', value: true },
-  { name: 'vaOnlineSchedulingCCDirectScheduling', value: false },
+  { name: 'vaOnlineSchedulingCCDirectScheduling', value: true },
```

Comment:
What is the rule here? My previous program the preference was to match what features are turned on in production.